core/templates/pages/exploration-editor-page/services/exploration-property.service.ts

```diff
@@ -155,6 +155,13 @@ export class ExplorationPropertyService {
     this._explorationPropertyChangedEventEmitter.emit();
   }
 
+  saveDisplayedValueAsync(): Promise<void> {
```

Comment:
Don't we need to write a test case for this?

Reply:
Done